app/controllers/api/base_controller/renderer.rb

```diff
@@ -666,6 +666,13 @@ def status_from_resource(resource)
           :bad_request
         end
       end
+
+      # currently, this is only called if it has a value
```

Comment:
What does this comment mean?

I'm hesitant commenting about callers at the call site cause if we add a new caller somewhere else, then this comment becomes Basmati Rice.